varcode/util.py

```diff
@@ -28,7 +28,11 @@
 _transcript_ids_cache = {}
 
 
-def random_variants(count, ensembl_release=MAX_ENSEMBL_RELEASE):
+def random_variants(
```

Comment:
consider setting a random seed (or taking one as an optional argument) so that the tests are deterministic? or does `random` module default to a fixed seed?